Modules/Loadable/Segmentations/MRMLDM/vtkMRMLSegmentationsDisplayableManager3D.cxx

```diff
@@ -71,15 +81,31 @@ class vtkMRMLSegmentationsDisplayableManager3D::vtkInternal
       this->Actor->SetMapper(mapper.GetPointer());
       this->Actor->SetVisibility(false);
 
+      this->CapActor = vtkSmartPointer<vtkActor>::New();
```

Comment:
Do we want to incorporate a revamp of the model DM in this PR, or later?

Reply:
Since it is an internal implementation detail, it probably makes sense to make this available for users with the current implementation. We will revamp the code (and incorporate other potential fixes and improvements based on user feedback) in a follow-up pull request.